core/wiki/macros/functions.tid

```diff
@@ -0,0 +1,5 @@
+title: $:/core/macros/functions
+tags: $:/tags/Global
+
+<!-- This global function can be used when the eexplicitState parameter is used with tabs-macros -->
+\function tf.getExplicitState() [<currentTab>get[explicitState]] ~[<currentTiddler>get[explicitState]] ~[<thisTiddler>get[explicitState]]
```

Comment:
Why is this cascade needed? It feels like it could very confusing.

I don't like having to introduce such a specific core function. It doesn't seem particularly useful outside the core.

Reply:
>Why is this cascade needed? It feels like it could very confusing.

It is necessary. Consider the following configuration. The story river contains 

- [$ControlPanel](https://tiddlywiki5-git-fork-pmario-explicit-f880f8-jermolenes-projects.vercel.app/#%24%3A%2FControlPanel) -> Info -> Cascades
- [$:/core/ui/ControlPanel/Cascades](https://tiddlywiki5-git-fork-pmario-explicit-f880f8-jermolenes-projects.vercel.app/#%24%3A%2Fcore%2Fui%2FControlPanel%2FCascades)
- "New Tiddler" with this code: `{{||$:/core/ui/ControlPanel/Cascades}}`

With the code in the PR all 3 tabs show the same "selected tab" -- all visible tiddlers can be used to switch tabs. All of them use the right explicitState value.

If `~[<thisTiddler>get[explicitState]]` is missing from the cascade, the "New Tiddler" will be **disconnected**, which is not what we want.

Reply:
>I don't like having to introduce such a specific core function. It doesn't seem particularly useful outside the core.

At the moment this "function" is a bit "lonely". But I would like to be able to do this: 

`<<switch-to-tab root:"$:/ControlPanel" path:"Info/Cascades/View Template Title">>` and the tab should be selected in the $:/ControlPanel tiddler. 

Every tiddler that contains a "tabs" macro will probably need more meta-data, but the functionality will be globally useful for everyone, that uses the tabs-tiddlers with that configuration.

Reply:
So after calling an action or a button it should look like this:

![image](https://github.com/user-attachments/assets/c6700df5-1d43-46cc-b2ce-6b5126eb7bd8)